pkg/controller/controller_instance_test.go

```diff
@@ -524,6 +560,8 @@ func TestReconcileInstance(t *testing.T) {
 	}
 }
 
+// TestReconcileInstanceAsynchronous tests provisioning a new service where the request results in a async response.
+// Resulting status will indicate not ready and polling in progress.
 func TestReconcileInstanceAsynchronous(t *testing.T) {
 	key := osb.OperationKey(testOperation)
 	fakeKubeClient, fakeCatalogClient, fakeBrokerClient, testController, sharedInformers := newTestController(t, fakeosb.FakeClientConfiguration{
```

Comment:
Nevermind, it does, but the kube and catalog client checks are interspersed - can you fix that please?  (it is my fault :))